src/System.Reflection.Metadata/src/System.Reflection.Metadata.builds

```diff
@@ -3,6 +3,9 @@
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
   <ItemGroup>
     <Project Include="System.Reflection.Metadata.csproj" />
+    <Project Include="System.Reflection.Metadata.csproj">
+      <AdditionalProperties>MetadataWriter=true</AdditionalProperties>
```

Comment:
One thing you might consider since you chose a more generic name (future) is to just make that a known target group to cross-compile by in dir.props.  That will allow you to remove the output path properties in the project.